nautobot_chatops/dispatchers/slack.py

```diff
@@ -204,7 +204,7 @@ def send_blocks(self, blocks, callback_id=None, ephemeral=False, modal=False, ti
             self.send_exception(slack_error)
 
     @BACKEND_ACTION_SNIPPET.time()
-    def send_snippet(self, text):
+    def send_snippet(self, text, title=None):
```

Comment:
When we add new parameters to a dispatcher API function, the parameter should be added to this API for all of the different dispatcher classes, even if it's unused by the other dispatchers at present.

Reply:
@glennmatthews I went ahead and added the title parameter to the other dispatcher classes.